bunsen/bunsen-avro/src/main/java/com/cerner/bunsen/avro/converters/DefinitionToAvroVisitor.java

```diff
@@ -743,6 +743,8 @@ private static String lowercase(String string) {
 
   @Override
   public int getMaxDepth(String elementTypeUrl, String path) {
+    // should be an odd number!
```

Comment:
should we add more description and implications like we discussed around the number of answers and performance impact as well?

Reply:
Yes we should. I will do that as part of fixing #757 as I need to better understand the implications myself.